package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "developer-license-donotus",
+  "version": "1.0.0",
+  "description": "Lorem ipsum...",
```

Comment:
can we copy in the description you have in the readme here?

Reply:
this stuff is all autogenerated for node projects